pdns/dnsdistdist/Makefile.am

```diff
@@ -23,21 +23,24 @@ EXTRA_DIST=dnslabeltext.rl \
 	   .version \
 	   contrib
 
-bin_PROGRAMS = dnsdist
+bin_PROGRAMS = dnsdist test-dnsdist_cc
```

Comment:
Maybe we should add a `--enable-tests` switch to the `configure` options, as we do with `--enable-unit-tests` for the testrunner.